selene/schema/cpes/fields.py

```diff
@@ -94,6 +97,12 @@ def resolve_cve_refs(root, _info):
                 return cves
         return None
 
+    def resolve_deprecated_by(root, _info):
+        cpe_item = root.find('cpe/raw_data/{*}cpe-item')
+        if cpe_item is not None:
+            return cpe_item.get('deprecated_by')
+        return None
```

Comment:
This seems to be a common pattern now. Maybe we should add a `get_attribute_from_element(element, element_name, attribute_name)` function to the utils module?

Reply:
Good Idea!